tests_config/raw-serializer.js

```diff
@@ -0,0 +1,17 @@
+// source: https://github.com/prettier/prettier/blob/ee2839bacbf6a52d004fa2f0373b732f6f191ccc/tests_config/raw-serializer.js
+'use strict';
+
+const RAW = Symbol.for('raw');
+
+module.exports = {
```

Comment:
this is cool! why do we want to use a custom serializer? so that we don't have any additional "noise" from the standard one?

Reply:
That’s correct! Otherwise, you’d get something like this in the `.snap` file:

```js
exports[`solidity.sol 1`] = `
"pragma solidity ^0.4.23;
pragma   solidity   ^0.4.23;

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
pragma solidity ^0.4.23;

pragma solidity ^0.4.23;

"
`;
```